DuckDuckGo/NetworkProtectionRootView.swift

```diff
@@ -33,11 +35,12 @@ struct NetworkProtectionRootView: View {
                                                            serverInfoObserver: AppDependencyProvider.shared.serverInfoObserver,
                                                            locationListRepository: locationListRepository)
     }
+
     var body: some View {
-        if AppDependencyProvider.shared.vpnFeatureVisibility.isPrivacyProLaunched() {
```

Comment:
If this check ever comes back false, this view will be empty. We don't need the check any more, so I've removed it.